backend/Makefile

```diff
@@ -92,7 +92,7 @@ SSH_SERVER_ALIVE_COUNT_MAX?=60
 # - consider adding -M and -S options on ssh command when needed
 # - add db/tunnel as a dependency for all targets so that a tunnel is automatically opened if not already
 db/tunnel:
-	$(eval endpoint=$(shell aws rds describe-db-cluster-endpoints --db-cluster-identifier corpora-${DEPLOYMENT_STAGE}-corpora-api | jq '.DBClusterEndpoints[] | select(.EndpointType | contains("WRITER")) | .Endpoint'))
+	$(eval endpoint=$(shell aws rds describe-db-cluster-endpoints --db-cluster-identifier corpora-${DEPLOYMENT_STAGE}-corpora-api | jq -r '.DBClusterEndpoints[] | select(.EndpointType | contains("WRITER")) | .Endpoint'))
```

Comment:
nit; actually kind of surprised the previously configuration didn't fail

Reply:
it would just be a quoted value instead, right? While I've never quoted commands, it works (in zsh):
```
$ "ls"
```
:)

Reply:
yeah but it's the endpoint, i.e. `corpora-"dev"-corpora-api.cluster-c81u9imopfwl.us-west-2.rds.amazonaws.com`